lib/handlers/quote/quote.ts

```diff
@@ -491,19 +495,41 @@ export class QuoteHandler extends APIGLambdaHandler<
               decimals: tokenIn.decimals.toString(),
               address: tokenIn.address,
               symbol: tokenIn.symbol!,
+              buyFeeBps: enableFeeOnTransferFeeFetching
```

Comment:
i would def make it a function and do if/else statements this is quite hard to read haha (i know what you are trying to do tho)

Reply:
Good point. Hard to read means it's prone to error. Also I think the copy paste here is not good. I addressed the feedback by creating private methods. Then re-run with the integ-test after re-deploying to personal aws account to ensure this works.